common/src/main/java/com/microsoft/identity/common/internal/activebrokerdiscovery/BrokerDiscoveryClientFactory.kt

```diff
@@ -61,20 +56,15 @@ class BrokerDiscoveryClientFactory {
          **/
         @JvmStatic
         fun setNewBrokerDiscoveryEnabled(isEnabled: Boolean){
```

Comment:
So, we will remove these once OneAuth removes the usage?
Are we asking them to remove the usage?

Reply:
I'll reach out to nick once we've release the next common.